i18n/zh-Hans.toml

```diff
@@ -75,3 +79,7 @@ close_quotation_mark = "」"
 about_feeds = "这是Web Feed，又称为Atom Feed，把现在的网址复制到新闻阅读器即可订阅本站文章。造访「About Feeds」来了解更多资讯。"
 visit_the_site = "造访网站"
 recent_posts = "近期文章"
+
+showcase = "示例"
+tutorial = "教程"
```

Comment:
❌ You should place i18n for specific use-cases (i.e. not for the entire tabi userbase) in your personal `i18n` files.

See [the docs](https://welpo.github.io/tabi/blog/faq-languages/#how-do-i-customise-or-override-a-specific-text-string-on-my-website).